graylog2_handler/__init__.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import graypy
+
+import openerp.tools as tools
+
+
+_logger = logging.getLogger(__name__)
+_logger.info("GELF_Handler module loading...")
+
+gelf_enabled = tools.config.get('gelf_enabled', False)
+_logger.info("GELF Enabled: %s", gelf_enabled)
```

Comment:
Since the module does not need explicit installation, and you might have it in your addons path and not want to used it, I would prefer it to be silent unless "activated". 
You can use debug level for the two log messages above.